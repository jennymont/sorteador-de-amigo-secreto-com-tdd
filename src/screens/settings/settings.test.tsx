import { render } from "@testing-library/react";
import { RecoilRoot } from "recoil";
import Settings from ".";

const mockNavigate = jest.fn()

jest.mock('react-router-dom', () => {
    return{
        useNavigate: () => mockNavigate
    }
})

describe('a pagina de configuração', () => {
    test('deve ser renderizada corretamente', () => {
       const {container} = render(
            <RecoilRoot>
                <Settings />
            </RecoilRoot>

        )

        expect(container).toMatchSnapshot()
    })
})