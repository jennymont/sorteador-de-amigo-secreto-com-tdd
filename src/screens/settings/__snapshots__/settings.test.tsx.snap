// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`a pagina de configuração deve ser renderizada corretamente 1`] = `
<div>
  <div
    class="card"
  >
    <section>
      <h2>
        Vamo começar
      </h2>
      <form>
        <div
          class="group-input-btn"
        >
          <input
            placeholder="Insira os nomes do participantes"
            type="text"
            value=""
          />
          <button
            disabled=""
          >
            Adicionar
          </button>
        </div>
        
      </form>
      <ul />
      <footer
        class="footer-settings"
      >
        <button
          class="button-footer"
          disabled=""
        >
          Iniciar Brincadeira
        </button>
      </footer>
    </section>
  </div>
</div>
`;
